src/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx

```diff
@@ -93,7 +93,10 @@ export const ArtistArtworkFilterRefetchContainer = createRefetchContainer(
   {
     artist: graphql`
       fragment ArtistArtworkFilter_artist on Artist
-        @argumentDefinitions(input: { type: "FilterArtworksInput" }) {
+        @argumentDefinitions(
+          includeBlurHash: { type: "Boolean!", defaultValue: true }
```

Comment:
There's a ton of these that need to be threaded through, _and_ with a `Boolean!` and `defaultValue` (which I chose to use `true`, as that's the current status quo) - due to the use of `@include(...)` in those low-level fragments - `GridItem` and `FlatGridItem`. Was a game of whack-a-mole w/ the Relay compiler.

Reply:
<3 relay compiler

Reply:
I actually didn't get it - why need to 'prop drill' all these variables, GraphQL variables should be global (hand-wave-y). Good tune-out/mechanical work